packages/docusaurus-1.x/examples/basics/pages/en/users.js

```diff
@@ -11,30 +11,14 @@ const CompLibrary = require('../../core/CompLibrary.js');
 
 const Container = CompLibrary.Container;
 
-// Add the "repoUrl" key to your siteConfig in order to see this component
-const AddYourCompanySection = ({siteConfig}) => {
-  if ((siteConfig.repoUrl || '').length === 0) {
-    return null;
-  }
-
-  const editUrl = `${siteConfig.repoUrl}/edit/master/website/siteConfig.js`;
-  return (
-    <>
-      <p>Are you using this project?</p>
-      <a href={editUrl} className="button">
-        Add your company
-      </a>
-    </>
-  );
-};
-
 class Users extends React.Component {
   render() {
     const {config: siteConfig} = this.props;
     if ((siteConfig.users || []).length === 0) {
       return null;
     }
 
+    const editUrl = `${siteConfig.repoUrl}/edit/master/website/siteConfig.js`;
```

Comment:
We better get rid of this `const` in this case, because if the repo url is not set, then the url will be wrong... Let's do this:

```js
<a href={`${siteConfig.repoUrl}/edit/master/website/siteConfig.js`}...
```